LoopKit/Notification/LoopNotificationCategory.swift

```diff
@@ -17,4 +17,5 @@ public enum LoopNotificationCategory: String {
     case pumpExpirationWarning
     case pumpExpired
     case pumpFault
+    case cgmAlert
```

Comment:
I notice that the pump notifications are fairly specific. Do we want to get as specific with cgm notifications?

Reply:
At this point, I'm not sure.